### PR TITLE
Attendance styling

### PR DESCRIPTION
**Pull request by sarahlauden**
This is a first draft of new attendance CSS for coloring and so-on. A new draft will be needed once Adam makes a few changes.

### CODE REVIEW

File: attendance.php [hunk 1]
```diff
@@ -754,21 +754,37 @@ function status_is_changeable_by_user($sta) {
 		display: inline-block;
 		padding: 0.5em 0.75em;
 		background-color: white;
+		border-color: #cecdcd;
+    border-style: solid;
```
Comment (sadleb) [inline attendance.php]: at some point we should set a consistent tabs/spaces rule and cleanup files to be consistent. i think you're using spaces and the existing file uses tabs. spaces are better in general, but this is about to get ugly real fast.

Comment (adamdruppe) [reply attendance.php]: tabs are objectively superior, spaces are ridiculous

but yeah it should just generally match whatever the rest of the file is doing

File: attendance.php [hunk 2]
```diff
@@ -957,6 +980,39 @@ function recordChange(ele, event_id, student_id, present) {
 	.main-attendance-view tbody tr:nth-child(2n + 1) {
 		background-color: #eee;
 	}
+
+	#save-button-holder input{
+	    background: #eb3b45;
+	    -webkit-border-radius: 5px;
```
Comment (adamdruppe) [inline attendance.php]: it doesn't hurt to have these, but the prefixed border radius things haven't been necessary for a long time now

Comment (sarahlauden) [reply attendance.php]: Yeah -- I saw you were using them in your website code so did it here as well.

File: attendance.php (hunk 1 above)
Comment (sarahlauden) [reply attendance.php]: Thanks @adamdruppe I'll be more intentional next time around this. 
hqdefault

**Pull request merged**